doc/CONTRIBUTING.md

```diff
@@ -73,6 +79,25 @@ If you already have a Python development environment on your machine and a prefe
 
 4. Tell virtualenvwrapper where on your machine to create virtual environments and add it to your profile. This is a one-time virtualenvwrapper setup step, and the process varies by operating system. [This tutorial](http://newcoder.io/begin/setup-your-machine/ "Python: setting up your computer") covers setting up virtualenvwrapper on OSX, Linux, and Windows.
    For Windows users, there may be extra steps needed.  If you run into an error on the import-module step, move the "VirtualEnvWrapper" folder from C:/Python27/Lib/site-packages/Users/*username*/Documents/WindowsPowerShell/Modules/ to C:/Users/*username*/Documents/WindowsPowerShell/Modules/.  Next, in powershell run the command "set-executionpolicy unrestricted".  Finally, in the VirtualEnvWrapper directory, open the file "VirtualEnvWrapperTabExpansion.psm1" and change "Function:TabExpansion" to "Function:TabExpansion2" in line 12.
+   
+   **Note:** After running:
```

Comment:
Yeah, it's a tough one. I can see this happening on Windows, but without access to a Windows machine can't verify that. Since your (terrific) directions are so specific to Mac, what do you think about updating **Note:** to **Note to Mac and Linux users:**?